_posts/2017-06-06-Spring-IO-2017-The-Spring-ecosystem.md

```diff
@@ -0,0 +1,318 @@
+A common example could be when shopping online. 
+The process has a clear starting and finishing view, but in between it can change state or views dynamically. 
+Through guided navigations, the user makes changes and it should register those changes as well as a possibility to finalise those change through a confirmation. 
+All this is possible with Spring Web Flow. 
+Although this project is listed with the main projects, there hasn't been any progress over the last years.
+
+## Spring Web Services
+
+<span class="image right small"><img class="p-image" alt="Spring Web Services" src="/img/spring-ecosystem/spring-web-services.png"></span>
+
+There are several ways to develop a web service, and one of which is used in combination with SOAP. 
+Spring Web Services helps with creating contract-first SOAP web services which are flexible by manipulating the XML contents. 
+But due to the popularity of the architectural style of REST, the interest in SOAP has diminished and that is noticeable in the maintenance of this Spring project which hasn't had any significant version updates. 
+
+# Popular
+
+When you look at modern applications and its infrastructure, you'll see the power of the Spring portfolio coming to its use. With the easy of use of Spring Boot, you can quickly start the development of a secure application and use Spring Cloud to help you with the deployment and integration for your online service provider. 
+
+<div class="row" style="margin: 0 auto 2.5rem auto;">
+<div class="col-md-offset-3 col-md-6">
+{% include image.html img="/img/spring-ecosystem/spring-popular.png" alt="Popular projects" title="Popular projects"%}
+</div>
+</div>
+
+A modern application might look like this:
+
+<div class="row" style="margin: 0 auto 2.5rem auto;">
+<div class="col-md-offset-3 col-md-6">
+{% include image.html img="/img/spring-ecosystem/modern-application.png" alt="Modern application" title="Modern application"%}
+</div>
+</div>
+
+## Spring Boot
+
+<span class="image left small"><img class="p-image" alt="Spring Boot" src="/img/spring-ecosystem/spring-boot.png"></span>
+
+Being built upon the Spring Framework, the popular Spring Boot provides an easy to use way for creating stand-alone Spring applications without code generation and configuration of XML files. 
+If you want to get started quickly without to much hassle, Spring Boot is the way to go by adding the modules you need. 
+Spring beans are not needed to be defined in XML, but are mostly automatically being configured by Spring Boot. 
+There's also so no need to find and configure libraries for your specific Spring version, Spring Boot tries to do that for you. 
+But if you wish, you can finetune it to your own needs by adding the library to the classpath of the application. 
+When you want to deploy your Spring Boot application there's no need to Web Application Archive file due to an embedded servers like Jetty, Tomcat and Undertow. 
```

Comment:
When you want to deploy your Spring Boot application, there's no need to build a WAR file, since you can build self-contained JAR files with an embedded servlet container such as Jetty, Tomcat and Undertow.